e2eTests/src/test/java/IngestionTest.java

```diff
@@ -242,20 +251,19 @@ private void verifyAfterFinalizeAndLookUpAccessionID() throws SQLException {
     log.info("Starting verification of state after finalize step...");
     String dbHost = "localhost";
     String port = "5432";
-    String db = "lega";
+    String db = "sda";
```

Comment:
Has the DB name changed in sda-db? Or is this a logical mix up with the sda exchange name for the broker?

Reply:
Yes, that's correct! The `db` name is indeed updated in the new repository. Here's the [old](https://github.com/neicnordic/sda-db/blob/7d921975f8b721ae34e9be117b4a43e730673618/entrypoint.sh#L37) and the [new](https://github.com/neicnordic/sensitive-data-archive/blob/e4e97ba915c073b4096e7431fc2a8bcf9f8bb362/postgresql/entrypoint.sh#L120) one for reference. We can still use `lega` as it is but they fallback to `sda` if no value is provided in the environment. So I thought of going with the standard names for this setup as well.